modules/virt-configuring-a-live-migration-policy.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * virt/live_migration/virt-configuring-live-migration-policies.adoc
+
+
+:_content-type: PROCEDURE
+[id="virt-configuring-a-live-migration-policy_{context}"]
+= Configuring a live migration policy
+
+Use the `MigrationPolicy` custom resource definition (CRD) to define migration policies for one or more groups of selected virtual machines (VMs).
+
+You can specify groups of VMs by using any combination of the following:
+* VM labels such as `size`, `os`, and `gpu`.
+* Namespace labels such as `priority`, `bandwidth`, and `hpc-workload`.
+
+[NOTE]
+====
+If multiple live migration policies apply to a VM, the policy with the highest number of matching labels will take precedence. If multiple policies meet this criteria, the policies will be sorted by lexicographic order of the matching labels keys, and the first one in order will take precedence.
+====
```

Comment:
Another important note is that all of the labels must match in order for a policy to math. IOW, if one label does not match the policy cannot be matched.